content/en/data_jobs/airflow.md

```diff
@@ -0,0 +1,197 @@
+</div>
+
+## Requirements
+
+* [Apache Airflow 2.8.0][1] or later is required.
+* [apache-airflow-providers-openlineage][2] 1.12.0 or later is required.
+
+## Setup
+
+Data Jobs Monitoring is supported for Apache Airflow deployment with [apache-airflow-providers-openlineage][2] installed.
+
+To get started, follow the instructions below.
+
+1. Ariflow provider `openlineage` is already installed in [Astro Runtimes][3]. Customize the provider version by adding the following to your `requirements.txt` file into your [Astro project][4].
+   
+   ```text
+   apache-airflow-providers-openlineage==<AIRLOW_OPENLINEAGE_PROVIDER_VERSION>
+   ```
+
+2. Configure `openlineage` provider. The simplest option is to set the following environment variables [using the Astro UI][5]:
+   
+   ```shell
+   OPENLINEAGE_URL=https://data-obs-intake.{{< region-param key="dd_site" code="true" >}}
+   OPENLINEAGE_ENDPOINT=api/v1/lineage
+   OPENLINEAGE_API_KEY=<DD_API_KEY>
+   ```
+
+   See [Astronomer official guide][5] for managing environment variables for a Deployment, and check official documentation [configuration-openlineage][6] for other supported configurations of `openlineage` provider.
+
+   **Important:**
+   * replace `<DD_API_KEY>` fully with your valid [Datadog API key][7].
+
+3. Trigger a update to your Deployment and wait for it to finish.
+
+[1]: https://github.com/apache/airflow/releases/tag/2.8.0
+[2]: https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/index.html
+[3]: https://www.astronomer.io/docs/astro/runtime-provider-reference
+[4]: https://www.astronomer.io/docs/astro/cli/develop-project
+[5]: https://www.astronomer.io/docs/astro/manage-env-vars#using-the-astro-ui
+[6]: https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/configurations-ref.html#configuration-openlineage
+[7]: https://docs.datadoghq.com/account_management/api-app-keys/#api-keys
+
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+## Requirements
+
+* [Apache Airflow 2.8.3][1] or later is required.
+* [apache-airflow-providers-openlineage 1.11.0][2] or later is required.
+
+## Setup
+
+Data Jobs Monitoring is supported for Apache Airflow deployment with [apache-airflow-providers-openlineage][2] installed.
+
+To get started, follow the instructions below.
+
+1. Install `openlineage` provider by adding the following into your `requirements.txt` file or wherever your Airflow depedencies is managed:
+   
+   ```text
+   apache-airflow-providers-openlineage>=1.11.0
+   ```
+
+2. Configure `openlineage` provider. The simplest option is to set the following environment variables and make them available to pods where you run Airflow schedulers and Airflow workers:
+   
+   ```shell
+   OPENLINEAGE_URL=https://data-obs-intake.{{< region-param key="dd_site" code="true" >}}
```

Comment:
It is not being substituted with the right value in the preview (showing
`<code class="js-region-param region-param" data-region-param="dd_site">`)